source/_integrations/rehau_nea_smart_manager.markdown

```diff
@@ -0,0 +1,64 @@
+---
+title: "Rehau Nea Smart Manager"
+description: ""
+ha_release: "0.0.1"
+ha_category: Climate
+ha_iot_class: "Local Polling"
+ha_quality_scale: internal
+ha_config_flow: true
```

Comment:
The description below does not suggest it is using a configuration flow but YAML configuration?